devimint/src/tests.rs

```diff
@@ -346,6 +346,8 @@ pub async fn latency_tests(dev_fed: DevFed, r#type: LatencyTest) -> Result<()> {
 }
 
 pub async fn cli_tests(dev_fed: DevFed) -> Result<()> {
+    let fedimint_cli_version = crate::util::FedimintCli::version_or_default().await;
+    let version_req = VersionReq::parse(">=0.3.0-alpha")?;
```

Comment:
If I see correctly, this variable is used only once, and having it here just makes it far from where it's used.

Reply:
I can move it down, was thinking that eventually all the tests will require version matching so define up top but nbd

Reply:
Fair enough. :+1: 

But... in that case it might need a longer name.